notebooks/8-rotation-speeds-pt2/interm_statistics.ipynb

```diff
@@ -0,0 +1,2490 @@
+{
```

Comment:
<p>This is good. I suspect you might be right about the number of columns, it could fall on the overwhelming side. Let's discuss including these, on the basis of NB length</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Disposition: if we can't fit into exercise, cut it out</p>

Reply:
<p>New disposition: sort these into "used" and "unused" statistics. People who really enjoy statistics can do the extra reading if they so choose</p>